modules/table/src/main/java/org/apache/ignite/internal/table/distributed/message/DataPresence.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.message;
+
+/**
+ * Whether a node has data or not (or it's not known).
+ */
+public enum DataPresence {
```

Comment:
Can we serialize it into an int, not a string?

Reply:
I don't mean to use ordinals, I mean to use an explicit int value. Don't we have a similar approach in other enums? Anyway, that's up to you

Reply:
Maybe I just missed this, but I did not see this approach in use for network messages. I still prefer strings for the reason I described earlier when the performance penalty is small (like in this case).